src/test/util/setup_common.cpp

```diff
@@ -218,6 +218,7 @@ void TestingSetup::LoadVerifyActivateChainstate()
     options.prune = chainman.m_blockman.IsPruneMode();
     options.check_blocks = m_args.GetIntArg("-checkblocks", DEFAULT_CHECKBLOCKS);
     options.check_level = m_args.GetIntArg("-checklevel", DEFAULT_CHECKLEVEL);
+    options.fail_on_insufficient_dbcache = false;
```

Comment:
In commit "init, validation: Improve handling if VerifyDB() fails due to insufficient dbcache" (cf307244db97e7b9e29a3d0ff948a6fa7877b548)

This block of code seems to duplicate a block of code in `AppInitMain`, but in the `AppInitMain` block this PR is setting `fail_on_insufficient_dbcache` to `args.IsArgSet("-checkblocks") || args.IsArgSet("-checklevel")`, while here it is being set to `false`. Probably both blocks of code should be deduplicated and call a shared function later. But for now it would be better to avoid creating unnecessary differences in the two blocks of code. Or if there is a reason test code is doing something different, there should be a code comment explaining why.

Reply:
Maybe just drop the hunk? Shouldn't matter either way :man_shrugging:

Reply:
changed to use same setting as in `AppInitMain`.